elao.app.docker/.manala/ansible/roles/release/tasks/main.yml

```diff
@@ -114,3 +114,47 @@
   args:
     chdir: "{{ release_dir }}"
   tags: log_failed
+
+# Next steps are pushing an empty commit to the original repository,
+# so that the release repository commit are easily identifiable.
+
+- name: git > Get release repo url
+  ansible.builtin.shell: >
+    git config --get remote.origin.url
+  args:
+    chdir: "{{ release_dir }}"
+  tags: log_failed
+  register: __release_repo_git_url
+
+- name: git > Get release repo informations
+  ansible.builtin.git:
+    repo: "{{ __release_repo_git_url.stdout }}"
+    dest: "{{ release_dir }}"
+    clone: false
+    update: false
+  register: __release_repo_git
+
+- name: git > Commit to original repository
+  ansible.builtin.shell: |
+    git {{
+      release_author|ternary(release_author|regex_replace(release_email_regex, '-c user.name="\g<name>" -c user.email="\g<email>"'), '')
+    }} commit --allow-empty --message "Released {{
+      ansible_date_time.year
+      ~ ansible_date_time.month
+      ~ ansible_date_time.day
+      ~ ansible_date_time.hour
+      ~ ansible_date_time.minute
+      ~ ansible_date_time.second
+    }}
+    Release repository commit: https://{{
+      __release_repo_git_url.stdout|regex_replace(release_git_url_regex, '\g<host>/\g<user>/\g<repository>')
+    }}/commit/{{ __release_repo_git.after }}"
+  tags: log_failed
+
+- name: git > Push to original repository
+  ansible.builtin.shell: >
+    git push --set-upstream origin
```

Comment:
No reference is passed after origin, it's using the current `HEAD` target. Might be an issue somehow ?

If someone is releasing locally but is currently on the wrong branch? That'll probably not be the only issue they have then.